src/mame/drivers/compc.cpp

```diff
@@ -213,8 +213,8 @@ ROM_START(pc10iii)
 	ROM_SYSTEM_BIOS(8, "v441", "v4.41")
 	ROMX_LOAD("318085-08.u201", 0x8000, 0x8000, CRC(7e228dc8) SHA1(958dfdd637bd31c01b949fac729d6973a7e630bc), ROM_BIOS(9))
 	ROM_REGION(0x8000, "gfx1", 0)
-	// ROM_LOAD("318086-02.u607", 0x0000, 0x8000, CRC(b406651c) SHA1(856f58353391a74a06ebb8ec9f8333d7d69e5fd6))
-	ROM_LOAD("318086-01_p10c_164a.bin", 0x0000, 0x4000, CRC(ee6c27f0) SHA1(e769cc3a49a1d708bd74eb4ac85bb6ea67220d38))
```

Comment:
Having the hash commented out is useless as the ROM won't end up in ROM sets - the BIOS feature works across multiple ROM regions, so you can load the correct chargen ROM for each BIOS.  See e.g. `osborne1` for an example.